src/clj_meowallet/http.clj

```diff
@@ -30,37 +32,68 @@
     (json/generate-string body)
     body))
 
+(defn get-status-from-error
+  [response]
+  (-> (.getMessage response)
+      (str/split #" ")
```

Comment:
How about turning of exceptions and [processing that yourself](https://github.com/clanhr/clanhr-api/blob/master/src/clanhr_api/core.clj#L208)? This way you'll have a response with a status.